packages/vscode-extension/src/webview/components/Preview.tsx

```diff
@@ -137,24 +137,37 @@ type TouchPointMarkerProps = {
   x: number;
   y: number;
   isPressing: boolean;
+  sizeFactor?: number;
+  color?: string;
+  pressedColor?: string;
 };
 
-function TouchPointMarker({ x, y, isPressing }: TouchPointMarkerProps) {
+function TouchPointMarker({
+  x,
+  y,
+  isPressing,
+  color = "175, 175, 175",
```

Comment:
I left another note in the place you use this component, but is it necessary to pass it here as a prop can't we achieve this with css?